Flicker/Screens/Artist/ArtistTappedViewController.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  ArtistTappedView.swift
+//  Flicker
+//
+//  Created by Jisu Jang on 2022/10/29.
+//
+import UIKit
+import SnapKit
+import Foundation
+
+class ArtistTappedViewController: UIViewController {
+
+    private let networkManager = NetworkManager.shared
+
+    private var posts: [Post] = []
+
+    private lazy var collectionView: UICollectionView = {
+        let collectionView = UICollectionView( frame: .zero, collectionViewLayout: UICollectionViewLayout())
+        let layout = UICollectionViewFlowLayout()
+//        layout.headerReferenceSize = CGSize(width: view.frame.width, height: HeaderCollectionReusableView().frame.height)
+        collectionView.collectionViewLayout = layout
+        collectionView.contentInsetAdjustmentBehavior = .never
+        return collectionView
+    }()
+
+    private let bottomBackgroundView = {
+        let UIView = UIView()
+        UIView.backgroundColor = .white
+        return UIView
+    }()
+
+    private let counselingButton = UIButton().then {
+        $0.setTitle("문의하기", for: .normal)
+        $0.titleLabel?.font = UIFont.preferredFont(forTextStyle: .callout, weight: .black)
+        $0.setTitleColor(.white, for: .normal)
+        $0.backgroundColor = .systemBlue
+        $0.layer.cornerRadius = 15
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad ()
+        print("ViewControlelr header \(HeaderCollectionReusableView().frame.height)")
+        collectionView.register(ArtistPortfolioCell.self, forCellWithReuseIdentifier: ArtistPortfolioCell.identifier)
+
+        collectionView.register(HeaderCollectionReusableView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: HeaderCollectionReusableView.identifier)
+
+        setDelegateAndDataSource()
+        view.addSubviews(collectionView, bottomBackgroundView, counselingButton)
+
+        queryImageDataSet()
+    }
+
+    private func setDelegateAndDataSource() {
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.showsVerticalScrollIndicator = false
+    }
+    
+    override func viewDidLayoutSubviews () {
+        super.viewDidLayoutSubviews()
+        collectionView.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.bottom.equalTo(view.safeAreaLayoutGuide).inset(50)
+            $0.centerX.centerY.equalToSuperview()
```

Comment:
```suggestion
            $0.center.equal.ToSuperview()
```

p2
로 바꾸시면 코드가 더 깔꼼해집니다

Reply:
꿀팁 감사합니당